1-exercises/E-for-loop/exercise1.js

```diff
@@ -4,11 +4,14 @@
     Change the while loop below into a for loop.
 */
 
-
 // Change the below code to use a for loop instead of a while loop.
 let i = 0;
-while(i < 26) {
-    console.log(String.fromCharCode(97 + i));
-    i++;
+// while (i < 26) {
+//   console.log(String.fromCharCode(97 + i));
+//   i++;
+// }
+
+for (i; i < 26; i++) {
```

Comment:
This bit of code raises a ReferenceError. 
What do you think is missing from the `for `statement?